composer.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "excelwebzone/recaptcha-bundle",
-    "description": "This bundle provides easy reCAPTCHA form field integration",
+    "name": "lethak/excelwebzonerecaptcha-bundle",
+    "description": "This bundle provides easy reCAPTCHA form field integration (forked from excelwebzone) ",
```

Comment:
You should just fork the orignal bundle and point an alias branch in your composer.json to your fork. 

Then no change on the package name is needed and you can use your branch without modified package name at all